Controller/ResourceController.php

```diff
@@ -123,17 +123,39 @@ protected function findOrThrowNotFound(ResourceInterface $resource, $id)
     protected function getCollectionData(ResourceInterface $resource, Request $request)
     {
         $page = (int) $request->get('page', 1);
-
+        $order = [];
         $filters = [];
-        foreach ($resource->getFilters() as $resourceFilter) {
+        $resourceFilters = $resource->getFilters();
+        $resourceOrders = $resource->getOrder();
```

Comment:
Minor but the variable must be named `$resourceOrder` or the method `getOrders()` for coherence.